google-cloud-datastore/src/main/java/com/google/cloud/datastore/spi/v1/DatastoreRpc.java

```diff
@@ -96,4 +96,9 @@ BeginTransactionResponse beginTransaction(BeginTransactionRequest request)
   default RunAggregationQueryResponse runAggregationQuery(RunAggregationQueryRequest request) {
     throw new UnsupportedOperationException("Not implemented.");
   }
+
+  @Override
+  default void close() throws Exception {
```

Comment:
why make this a default method? Should we just implement in all places?

Reply:
We can do that, but they will be empty too like this one.

Reply:
addressed in 4ed8259.